interface-definitions/container.xml.in

```diff
@@ -13,6 +13,23 @@
             <regex>[-a-zA-Z0-9]+</regex>
           </constraint>
           <constraintErrorMessage>Container name must be alphanumeric and can contain hyphens</constraintErrorMessage>
+          <childSpecification>
+            <requiredChildren>
+              <child>image</child>
+            </requiredChildren>
+            <mutuallyExclusiveChildren>
+              <child>allow-host-networks</child>
+              <child>network</child>
+            </mutuallyExclusiveChildren>
+            <atLeastOneOf>
+              <child>allow-host-networks</child>
+              <child>network</child>
+            </atLeastOneOf>
+            <oneWayDependantChildren>
```

Comment:
The word "dependent" is misspelled here (see https://en.wiktionary.org/wiki/dependent)

I also wonder if it should be just "dependentChildren" rather than "oneWayDependentChildren".